egs/callhome_diarization/v1/local/train_ivector_extractor_diag.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
```

Comment:
Why do we need this script? There should be a comment at the top describing that this is a copy of some original script, with various modifications for whatever reason. Also, this should probably go in diarization/ rather than local/ . @HuangZiliAndy can you also look at this please.